lib/LaTeXML/Core/Definition/Expandable.pm

```diff
@@ -19,20 +19,25 @@ use LaTeXML::Common::Error;
 use LaTeXML::Core::Token;
 use LaTeXML::Core::Tokens;
 use LaTeXML::Core::Parameters;
+use LaTeXML::Package qw(TokenizeInternal);
 use base qw(LaTeXML::Core::Definition);
 
 sub new {
   my ($class, $cs, $parameters, $expansion, %traits) = @_;
+  if (!ref $expansion) {    # Tokenization DEFERRED till actually used (shaves > 5%)
```

Comment:
I lost the 5% optimization actually, maybe I can re-gain it once everything is finished and one can rethink optimizing...